package.json

```diff
@@ -17,9 +17,16 @@
   "license": "MIT",
   "devDependencies": {
     "broccoli-asset-rev": "^2.4.5",
+    "ember-ajax": "^3.1.0",
```

Comment:
Which thing depends on this?

Reply:
`ember-cli-addon-docs` depends on it currently, but it's not currently listed as a dependency.

https://github.com/ember-learn/ember-cli-addon-docs/issues/76